src/components/Sum.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a, b }) => {
+  const sum = a + b;
+
+  return (
+    <p>
+      Sum of
+      {' '}
+      {a}
+      {' '}
+      and
+      {' '}
+      {b}
+      {' '}
+      is
+      {' '}
+      {sum}
```

Comment:
It can be shorter ;)
```suggestion
      {`Sum of ${a} and ${b} is ${sum}`}
```

Reply:
Agree with Daryna, you should fix it, it's really hard to read